services/api.py

```diff
@@ -1071,12 +1091,16 @@ def list(self, request, *args, **kwargs):
 
         # Switch between paginated or standard style responses
         page = self.paginate_queryset(self.object_list)
-        serializer = self.get_serializer(page, many=True)
-        resp = self.get_paginated_response(serializer.data)
-
-        translation.activate(old_language)
-
-        return resp
+        serializer = self.get_serializer(page, many=True, data=request.data)
+        print(serializer)
```

Comment:
Remove.

Reply:
This was left from testing and I think the check is not needed here. This part is 
restored to the original state.